opencog/benchmark/AtomSpaceBenchmark.cc

```diff
@@ -1178,7 +1178,7 @@ timepair_t AtomSpaceBenchmark::bm_getOutgoingSet()
 #if HAVE_CYTHONX
```

Comment:
should probably change this back to HAVE_CYTHON from HAVE_CYTHONX .. the X was added because the code was wonky and had to be disabled.